vite.config.ts

```diff
@@ -39,7 +39,12 @@ export default defineConfig(({ command }) =>
                       formats: ['es', 'umd', 'iife'],
                   },
                   rollupOptions: {
+                      external: ['vue', /^ol.*/],
                       output: {
+                          globals: {
+                              'ol/MapBrowserEvent': 'MapBrowserEvent',
```

Comment:
`'ol/MapBrowserEvent': 'ol.MapBrowserEvent'`

not tested with examples that import `ol-contextmenu`, could you do that, please?